components/LibraryIndex/LibraryIndex.jsx

```diff
@@ -16,7 +16,7 @@ function LibraryIndex() {
 	const allLibraryItems = libraryItems.map((libraryItem) => (
 		<ul key={libraryItem._id}>
 			<li>
-				<Link to={`library/${libraryItem._id}`}>Name: {libraryItem.name}</Link>
+				<Link to={`${libraryItem._id}`}>Name: {libraryItem.name}</Link>
```

Comment:
it should be {`/library/${libraryItem._id}`}

Reply:
Yeah I thought so too, but for some reason when I click the link from the index it ends up being:
/library/library

So I deleted one instance.